src/main/resources/templates/product/productDetails.html

```diff
@@ -165,6 +225,14 @@ <h4 class="mdl-dialog__title">Archive this product?</h4>
     dialog.querySelector('.close').addEventListener('click', function () {
         dialog.close();
     });
+
+    /* logic for displaying review form */
+    function showReviewForm() {
+        let formSection = document.querySelector(".review-form");
```

Comment:
there's only one div with review-form, so could be 'const'

Reply:
fixed